### PR TITLE
Fix bookmark clipping

### LINKED ISSUE #4614: Bookmark text is cut off in bookmarks toolbar
**Did you search for similar issues before submitting this one?**
Yes

**Describe the issue you encountered:**
Recent (0.12.4) changes have caused the line height or another attribute with the text in the bookmarks toolbar to be off.  The bottom portion of the letter g and other letters get cut off (see screenshot)

**Expected behavior:**
This shouldn't happen

- Platform (Win7, 8, 10? macOS? Linux distro?):

- Brave Version:

- Steps to reproduce:
    1. Use Brave 0.12.4
    2. Visit google.com
    3. Bookmark the site and ensure it shows in bookmarks toolbar
    4. It should look like the screenshot (bottom part of the g is cut off)

- Screenshot if needed:
screen shot 2016-10-07 at 12 52 40 pm

- Any related issues:
May have been caused by https://github.com/brave/browser-laptop/pull/4400 (font size changes I believe?). Needs investigation

### LINKED ISSUE #4635: Bookmarks are clipped at the bottom
**Did you search for similar issues before submitting this one?**
Yes. Did not find anything.

**Describe the issue you encountered:**
The titles of bookmarks on the bookmarks toolbar are clipped at the bottom. It is most obvious with characters that extend below the baseline, such as `q y p g j`. This problem occurs with favicons on or off.

It is possible that this problem was introduced after the UI changes in 0.12.4. A suggested fix is to increase the height of the bookmark items.

**Expected behavior:**
The bookmark titles should be completely visible and not clipped.

- Platform (Win7, 8, 10? macOS? Linux distro?):
macOS

- Brave Version:
0.12.4

- Steps to reproduce:
    1. Create a bookmark for any page.
    2. Set the title to `q y p g j`.
    3. Notice that the letters are clipped at the bottom.

- Screenshot if needed:
image

- Any related issues:
None.

### PR DESCRIPTION
**Pull request by Liunkae**
- [x] Submitted a [ticket](https://github.com/brave/browser-laptop/issues) for my issue if one did not already exist.
- [x] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
- [x] Added/updated tests for this change (for new code or code which already has tests).
- [x] Ran `git rebase -i` to squash commits (if needed).

Test Plan:
1. Create a bookmark for any page.
2. Set the title to `q y p g j`.
3. Make sure the letters are not clipped at the bottom.

Fixes #4614

### CODE REVIEW

**Comment by Liunkae** [comment]
Fixes #4614

File: less/bookmarksToolbar.less [hunk 1]
```diff
@@ -44,7 +44,7 @@
     color: black;
     cursor: default;
     font-size: 11px;
-    line-height: 11px;
+    line-height: 14px;
```
Comment (bbondy) [inline less/bookmarksToolbar.less]: This looks great when bookmark toolbar icons is turned on but when there is not it throws off the vertical alignment a lot.  Could you fix? Thanks!

**Comment by Liunkae** [comment]
I decided to reduce the line height of the bookmark button and increase the toolbar height. I hope this is the right approach.

File: less/bookmarksToolbar.less (hunk 1 above)
Comment (Liunkae) [inline less/bookmarksToolbar.less]: Please try the new commit (d289f1a).

**Comment by bbondy** [comment]
++

**Pull request merged**